.travis.yml

```diff
@@ -16,9 +16,8 @@ install:
 jdk: oraclejdk8
 env:
   matrix:
-    # ECLIPSE_TARGET is the targeted platform: mars is the default target
+    # ECLIPSE_TARGET is the targeted platform: oxygen is the default target
     # JAVA_RUNTIME specifies the execution runtime for tests; must match an <id> in .travisci/toolchains.xml
-    - JAVA_RUNTIME=JavaSE-1.7
     - JAVA_RUNTIME=JavaSE-1.8 ECLIPSE_TARGET=neon
     - JAVA_RUNTIME=JavaSE-1.8 ECLIPSE_TARGET=oxygen MAVEN_FLAGS=-Pjacoco
```

Comment:
Need to remove `ECLIPSE_TARGET=oxygen`, since that's the default.

Reply:
And need to remove `-Declipse.target=oxygen` from `kokoro/windows/continuous.bat`.